<?php

return [
    "Divisions have been retrieved successfully" => "تم استرداد الإدارات بنجاح",
    "Division has been retrieved successfully" => "تم استرداد الإدارة بنجاح",
    "Division has been created successfully" => "تم إنشاء الإدارة بنجاح",
    "Division has been updated successfully" => "تم تحديث الإدارة بنجاح",
    "Division has been deleted successfully" => "تم حذف الإدارة بنجاح",
    "Error happened while retrieving divisions" => "حدث خطأ أثناء الحصول على الإدارات",
    "Error happened while retrieving division" => "حدث خطأ أثناء الحصول على بيانات الإدارة",
    "Error happened while storing division" => "حدث خطأ أثناء تخزين الإدارة",
    "Error happened while deleting division" => "حدث خطأ أثناء حذف الإدارة",
    "Division has been restored successfully" => "تمت استعادة الإدارة بنجاح",
    "Error happened while restoring division" => "حدث خطأ أثناء استعادة الإدارة",
    "Division has been force deleted successfully" => "تم حذف الإدارة بشكل نهائي بنجاح",
    "Error happened while force deleting division" => "حدث خطأ أثناء حذف الإدارة بشكل نهائي",
    "Division has not deleted before" => "لم يتم حذف القسم من قبل",

    "Departments have been retrieved successfully" => "تم استرداد الأقسام بنجاح",
    "Department has been retrieved successfully" => "تم استرداد القسم بنجاح",
    "Department has been created successfully" => "تم إنشاء القسم بنجاح",
    "Department has been updated successfully" => "تم تحديث القسم بنجاح",
    "Department has been deleted successfully" => "تم حذف القسم بنجاح",
    "Error happened while retrieving departments" => "حدث خطأ أثناء الحصول على الأقسام",
    "Error happened while retrieving department" => "حدث خطأ أثناء الحصول على بيانات القسم",
    "Error happened while storing department" => "حدث خطأ أثناء تخزين القسم",
    "Error happened while deleting department" => "حدث خطأ أثناء حذف القسم",
    "Department has been restored successfully" => "تمت استعادة القسم بنجاح",
    "Error happened while restoring department" => "حدث خطأ أثناء استعادة القسم",
    "Department has been force deleted successfully" => "تم حذف القسم بنجاح",
    "Error happened while force deleting department" => "حدث خطأ أثناء حذف القسم بشكل نهائي",
    "Department has not deleted before" => "لم يتم حذف القسم من قبل",

    "Positions have been retrieved successfully" => "تم استرداد الوظائف بنجاح",
    "Position has been retrieved successfully" => "تم استرداد الوظيفة بنجاح",
    "Position has been created successfully" => "تم إنشاء الوظيفة بنجاح",
    "Position has been updated successfully" => "تم تحديث الوظيفة بنجاح",
    "Position has been deleted successfully" => "تم حذف الوظيفة بنجاح",
    "Error happened while retrieving positions" => "حدث خطأ أثناء الحصول على الوظائف",
    "Error happened while retrieving position" => "حدث خطأ أثناء الحصول على بيانات الوظيفة",
    "Error happened while storing position" => "حدث خطأ أثناء تخزين الوظيفة",
    "Error happened while deleting position" => "حدث خطأ أثناء حذف الوظيفة",
    "Position has been restored successfully" => "تمت استعادة الوظيفة بنجاح",
    "Error happened while restoring position" => "حدث خطأ أثناء استعادة الوظيفة",
    "Position has been force deleted successfully" => "تم حذف الوظيفة بنجاح",
    "Error happened while force deleting position" => "حدث خطأ أثناء حذف الوظيفة بشكل نهائي",
    "Position has not deleted before" => "لم يتم حذف الموقف من قبل",

    "Industries have been retrieved successfully" => "تم استرداد الصناعات بنجاح",
    "Industry has been retrieved successfully" => "تم استرداد الصناعة بنجاح",
    "Industry has been created successfully" => "تم إنشاء الصناعة بنجاح",
    "Industry has been updated successfully" => "تم تحديث الصناعة بنجاح",
    "Industry has been deleted successfully" => "تم حذف الصناعة بنجاح",
    "Error happened while retrieving industries" => "حدث خطأ أثناء الحصول على الصناعات",
    "Error happened while retrieving industry" => "حدث خطأ أثناء الحصول على بيانات الصناعة",
    "Error happened while storing industry" => "حدث خطأ أثناء تخزين الصناعة",
    "Error happened while deleting industry" => "حدث خطأ أثناء حذف الصناعة",
    "Industry has been restored successfully" => "تمت استعادة الصناعة بنجاح",
    "Error happened while restoring industry" => "حدث خطأ أثناء استعادة الصناعة",
    "Industry has been force deleted successfully" => "تم حذف الصناعة بنجاح",
    "Error happened while force deleting industry" => "حدث خطأ أثناء حذف الصناعة بشكل نهائي",
    "Industry has not deleted before" => "لم يتم حذف الصناعة من قبل",

    "Titles have been retrieved successfully" => "تم استرداد الألقاب بنجاح",
    "Title has been retrieved successfully" => "تم استرداد اللقب بنجاح",
    "Title has been created successfully" => "تم إنشاء اللقب بنجاح",
    "Title has been updated successfully" => "تم تحديث اللقب بنجاح",
    "Title has been deleted successfully" => "تم حذف اللقب بنجاح",
    "Error happened while retrieving titles" => "حدث خطأ أثناء الحصول على الألقاب",
    "Error happened while retrieving title" => "حدث خطأ أثناء الحصول على بيانات اللقب",
    "Error happened while storing title" => "حدث خطأ أثناء تخزين اللقب",
    "Error happened while deleting title" => "حدث خطأ أثناء حذف اللقب",
    "Title has been restored successfully" => "تمت استعادة اللقب بنجاح",
    "Error happened while restoring title" => "حدث خطأ أثناء استعادة اللقب",
    "Title has been force deleted successfully" => "تم حذف اللقب بنجاح",
    "Error happened while force deleting title" => "حدث خطأ أثناء حذف اللقب بشكل نهائي",
    "Title has not deleted before" => "لم يتم حذف اللقب من قبل",

    "Companies have been retrieved successfully" => "تم استرداد الشركات بنجاح",
    "Company has been retrieved successfully" => "تم استرداد الشركة بنجاح",
    "Company has been created successfully" => "تم إنشاء الشركة بنجاح",
    "Company has been updated successfully" => "تم تحديث الشركة بنجاح",
    "Company has been deleted successfully" => "تم حذف الشركة بنجاح",
    "Error happened while retrieving companies" => "حدث خطأ أثناء استرجاع الشركات",
    "Error happened while retrieving company" => "حدث خطأ أثناء استرداد الشركة",
    "Error happened while storing company" => "حدث خطأ أثناء تخزين الشركة",
    "Error happened while deleting company" => "حدث خطأ أثناء حذف الشركة",
    "Company has been restored successfully" => "تمت استعادة الشركة بنجاح",
    "Error happened while restoring company" => "حدث خطأ أثناء استعادة الشركة",
    "Company has been force deleted successfully" => "لقد تم حذف الشركة بشكل نهائي بنجاح",
    "Error happened while force deleting company" => "حدث خطأ أثناء حذف الشركة بشكل نهائي",
    "Company has not deleted before" => "لم يتم حذف الشركة من قبل",

    "Contacts has been retrieved successfully" => "تم استرداد جهات الاتصال بنجاح",
    "Contacts has been updated successfully" => "تم تحديث جهات الاتصال بنجاح",
    "Contact has been deleted successfully" => "تم حذف جهة الاتصال بنجاح",
    "Error happened while retrieving contacts" => "حدث خطأ أثناء استرداد جهات الاتصال",
    "Error happened while updating contact" => "حدث خطأ أثناء تحديث جهات الاتصال",
    "Error happened while deleting contact" => "حدث خطأ أثناء حذف جهة الاتصال",
    "Contact has been restored successfully" => "تمت استعادة جهة الاتصال بنجاح",
    "Error happened while restoring contact" => "حدث خطأ أثناء استعادة جهة الاتصال",
    "Contact has been force deleted successfully" => "لقد حذف جهة الاتصال بشكل نهائي بنجاح",
    "Error happened while force deleting contact" => "حدث خطأ أثناء حذف جهة الاتصال بشكل نهائي",
    "Contact has not deleted before" => "لم يتم حذف جهة الاتصال من قبل",
    "Contact type has been checked successfully" => "تم التحقق من نوع جهة الاتصال بنجاح",

    "Notes has been retrieved successfully" => "تم استرداد الملاحظات بنجاح",
    "Note has been created successfully" => "تم إنشاء الملاحظة بنجاح",
    "Note has been deleted successfully" => "تم حذف الملاحظة بنجاح",
    "Error happened while retrieving notes" => "حدث خطأ أثناء استرداد الملاحظات",
    "Error happened while storing note" => "حدث خطأ أثناء تخزين الملاحظة",
    "Error happened while updating note" => "حدث خطأ أثناء تحديث الملاحظة",
    "Error happened while deleting note" => "حدث خطأ أثناء حذف الملاحظة",
    "Note has been restored successfully" => "تمت استعادة الملاحظة بنجاح",
    "Error happened while restoring note" => "حدث خطأ أثناء استعادة الملاحظة",
    "Note has been force deleted successfully" => "تم حذف الملاحظة بنجاح",
    "Error happened while force deleting note" => "حدث خطأ أثناء حذف الملاحظة بشكل نهائي",
    "Note has not deleted before" => "لم يتم حذف الملاحظة من قبل",

    "Clients have been retrieved successfully" => "تم استرداد العملاء بنجاح",
    "Client has been retrieved successfully" => "تم استرداد العميل بنجاح",
    "Client has been created successfully" => "تم إنشاء العميل بنجاح",
    "Client has been updated successfully" => "تم تحديث العميل بنجاح",
    "Client has been deleted successfully" => "تم حذف العميل بنجاح",
    "Error happened while retrieving clients" => "حدث خطأ أثناء استرداد العملاء",
    "Error happened while retrieving client" => "حدث خطأ أثناء استرداد العميل",
    "Error happened while storing client" => "حدث خطأ أثناء تخزين العميل",
    "Error happened while deleting client" => "حدث خطأ أثناء حذف العميل",
    "Client has been restored successfully" => "تمت استعادة العميل بنجاح",
    "Error happened while restoring client" => "حدث خطأ أثناء استعادة العميل",
    "Client has been force deleted successfully" => "تم فرض حذف العميل بنجاح",
    "Error happened while force deleting client" => "حدث خطأ أثناء حذف العميل بشكل نهائي",
    "Client has not deleted before" => "لم يتم حذف العميل من قبل",

    "Resource not found" => "غير موجود",

    "Client Interests have been retrieved successfully" => "تم استرداد اهتمامات العميل بنجاح",
    "Client Interest has been updated successfully" => "تم تعديل اهتمامات العميل بنجاح",
    "Client Interest has been retrieved successfully" => "تم استرداد اهتمامات العميل بنجاح",
    "Client Interest has been deleted successfully" => "تم حذف اهتمامات العميل بنجاح",
    "Client Interest has been restored successfully" => "تمت استعادة اهتمامات العميل بنجاح",
    "Client Interest has been force deleted successfully" => "لقد تم حذف اهتمامات العميل بشكل نهائي",

    "Error happened while storing interest" => "حدث خطأ أثناء تخزين اهتمامات العميل",
    "Error happened while retrieving interest" => "حدث خطأ أثناء استرداد اهتمامات العميل",
    "Error happened while deleting interest" => "حدث خطأ أثناء حذف اهتمامات العميل",
    "Error happened while restoring interest" => "حدث خطأ أثناء استعادة اهتمامات العميل",
    "Error happened while force deleting interest" => "حدث خطأ أثناء فرض حذف اهتمامات العميل",

    "Categories have been retrieved successfully" => "تم استرداد الفئات بنجاح",
];
